### PR TITLE
docs(linux): Add build doc for Keyman Web and Android

### PR DESCRIPTION
**Pull request by ermshiperete**
This adds the documentation how to build Keyman Web as well as Keyman for Android on Linux. and adds the dependencies to the `Dockerfile` so that it's now also possible to build Keyman Web and Keyman for Android with Docker.

@keymanapp-test-bot skip

### CODE REVIEW

File: docs/build/linux-ubuntu.md [hunk 1]
```diff
@@ -191,3 +206,59 @@ older versions, you can set `JAVA_HOME_11` to the OpenJDK 11 path and
 from command line. But note that you do need to update your `JAVA_HOME` env
 var to the associated version before opening Android Studio and loading any
 Android projects. `JAVA_HOME_11` is mostly used by CI.
+
+## Docker Builder
```
Comment (srl295) [inline docs/build/linux-ubuntu.md:210]: good, should we _remove_ the other discussion of this Dockerfile then?

**Review by srl295 — APPROVED** [review]

**Review by jahorton — APPROVED** [review]

File: docs/build/linux-ubuntu.md [hunk 2]
```diff
@@ -50,7 +51,7 @@ sudo apt install devscripts equivs
 sudo mk-build-deps --install linux/debian/control
 ```
 
-### Node.js
+#### Node.js
 
 Node.js v18 is required for Core build, Web tests, and Developer command line tools.
```
[the remark this answers is not in the document]
Comment (mcdurdin) [reply docs/build/linux-ubuntu.md:56]: This info also needs to migrate into the build docs for macOS and Windows.

File: docs/build/linux-ubuntu.md (hunk 1 above)
Comment (ermshiperete) [reply docs/build/linux-ubuntu.md:210]: Which other discussion do you mean?

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.152-alpha](https://keyman.com/downloads/releases/alpha/17.0.152)

**Pull request merged**